src/constants/preferences.ts

```diff
@@ -34,7 +34,6 @@ const Preferences: Dictionary<any> = {
     MENTION_KEYS: 'mention_keys',
     USE_MILITARY_TIME: 'use_military_time',
     CATEGORY_SIDEBAR_SETTINGS: 'sidebar_settings',
-    CHANNEL_SIDEBAR_ORGANIZATION: 'channel_sidebar_organization',
```

Comment:
Can you leave this constant? I still need it for the backwards compatibility code